src/js/ripple/amount.js

```diff
@@ -180,15 +190,19 @@ Amount.prototype.divide = function(divisor) {
  * @this {Amount} The numerator (top half) of the fraction.
  * @param {Amount} denominator The denominator (bottom half) of the fraction.
  * @param opts Options for the calculation.
- * @param opts.reference_date {Date|Number} Date based on which demurrage/interest
- *   should be applied. Can be given as JavaScript Date or int for Ripple epoch.
+ * @param opts.reference_date {Date|Number} Date based on which
+ * demurrage/interest should be applied. Can be given as JavaScript Date or int
+ * for Ripple epoch.
  * @return {Amount} The resulting ratio. Unit will be the same as numerator.
  */
 
 Amount.prototype.ratio_human = function(denominator, opts) {
   opts = extend({ }, opts);
 
+  /*eslint-disable consistent-this */
   var numerator = this;
```

Comment:
I think there's good reason to keep the existing `numerator` semantics. There is nothing to gain by changing it to `self`. Same for the `ref` thing below